features/home/impl/src/test/kotlin/io/element/android/features/home/impl/HomePresenterTest.kt

```diff
@@ -145,21 +148,50 @@ class HomePresenterTest {
         }
     }
 
-    private fun TestScope.createHomePresenter(
+    @Test
+    fun `present - NavigationBar is hidden when the last space is left`() = runTest {
+        val homeSpacesPresenter = MutablePresenter(aHomeSpacesState())
```

Comment:
I first reinvented `MutablePresenter` and see that it was already a thing, but not used (yet!)